src/ipred.rs

```diff
@@ -2206,6 +2282,9 @@ impl Rav1dIntraPredDSPContext {
     }
 
     pub const fn new<BD: BitDepth>(flags: CpuFlags) -> Self {
-        Self::default::<BD>().init::<BD>(flags)
+        let default = Self::default::<BD>();
+        let mut c = Self::default::<BD>().init::<BD>(flags);
+        c.intra_pred[FILTER_PRED as usize] = default.intra_pred[FILTER_PRED as usize];
```

Comment:
What's this change for? It seems like it's unrelated to the rest of the changes in the PR, and it's unclear why this is even necessary. This seems like it's overriding the value of `c.intra_pred[FILTER_PRED]` set in `init` with the default value, i.e. the Rust fallback. Is that the correct thing to do? Isn't that going to override the asm routine that was set in `init`?

Reply:
Whoops, I added this for debugging/checking I didn't break anything while running tests faster.  I did not mean to commit this.

Reply:
Fixed in [`c8c8921..cdc3c5e`](https://github.com/memorysafety/rav1d/compare/c8c89218a1ffb4537d12bf56e463d5f04c5b039d..cdc3c5e32b4e84bd0da00dca204bea2b740d1a60).